traffic_portal/bower.json

```diff
@@ -1,23 +1,17 @@
 {
     "name": "traffic_portal",
     "dependencies": {
-        "angular": "1.7.8",
-        "angular-animate": "1.7.8",
         "angular-resource": "1.7.8",
```

Comment:
by leaving these other angular ones in bower.json, its still pulling angular in via `bower install`

```
angular-bootstrap#0.14.3 app/bower_components/angular-bootstrap
└── angular#1.7.8
```

so i would suggest either a) limiting this PR to just removing unused deps (es5-shim and json3) or b) moving ALL of the angular* deps to npm.

Reply:
Not all of "the angular* deps" can be moved over so trivially. Particularly, the bootstrap stuff doesn't load properly when included with browserify from npm. I don't know why, but it might be related to the fact that the version we use from bower is unavailable in npm. Also note, though, that neither available versions are supported, neither package manager offers supported versions, and the version we use is two major versions behind what's current.

With all that in mind I was hoping to just remove angular-bootstrap* entirely, especially if it's only used for the table context menus, since the new ag-grid tables won't use angular-bootstrap-contextmenu.